### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by Dgohel11**

### CODE REVIEW

File: flaskapp/blueprints/courses/forms.py [hunk 1]
```diff
@@ -14,6 +14,15 @@
 
 
 def validate_course_name(form, course_name):
+    """Valiadtion on course name
```
Comment (Niraj-Kamdar) [inline flaskapp/blueprints/courses/forms.py:17]:
```suggest
    """Validation of course name
```

File: flaskapp/blueprints/courses/forms.py [hunk 2]
```diff
@@ -14,6 +14,15 @@
...
+
+    Args:
+        form (Object): In which given description abour course
```
Comment (Niraj-Kamdar) [inline flaskapp/blueprints/courses/forms.py:20]: about

File: flaskapp/blueprints/courses/forms.py [hunk 3]
```diff
@@ -23,6 +32,15 @@ def validate_course_name(form, course_name):
 
 
 def validate_unit_name(form, unit_name):
+    """Validation on unit name of the course 
+
+    Args:
+        form (Object): Forn i which all the details of unit
```
Comment (Niraj-Kamdar) [inline flaskapp/blueprints/courses/forms.py:38]: form

File: flaskapp/blueprints/courses/routes.py [hunk 4]
```diff
@@ -67,6 +67,8 @@ def all_courses():
 @courses.route("/course/delete/", methods=["GET", "POST"])
 @login_required
 def delete_course():
+    """Delete course from list of user
```
Comment (Niraj-Kamdar) [inline flaskapp/blueprints/courses/routes.py:70]: delete course from list of courses

File: flaskapp/blueprints/courses/routes.py [hunk 5]
```diff
@@ -78,6 +80,14 @@ def delete_course():
 @login_required
 @check_valid_course
 def all_units(course_id):
+    """Give all units list of course of given course ID
```
Comment (Niraj-Kamdar) [inline flaskapp/blueprints/courses/routes.py:83]: with given ....

File: flaskapp/blueprints/courses/forms.py [hunk 6]
```diff
@@ -14,10 +14,10 @@
 
 
 def validate_course_name(form, course_name):
-    """Valiadtion on course name
+    """Valiadtion of course name
```
Comment (Niraj-Kamdar) [inline flaskapp/blueprints/courses/forms.py:17]: still typo

File: flaskapp/blueprints/courses/routes.py [hunk 7]
```diff
@@ -80,7 +80,7 @@ def delete_course():
 @login_required
 @check_valid_course
 def all_units(course_id):
-    """Give all units list of course of given course ID
+    """With given all units list of course of given course ID
```
Comment (Niraj-Kamdar) [inline flaskapp/blueprints/courses/routes.py:83]: what?

**Comment by Niraj-Kamdar** [comment]
Looks great, you have to fix conflict so that it can be merged

**Comment by Niraj-Kamdar** [comment]
No problem I have solved it and merged it via PR #190

**Comment by Dgohel11** [comment]
Thank you.

**Pull request closed without merging**